deployment_scripts/config

```diff
@@ -18,8 +18,4 @@ WINDOWS_64_BINARY_NAME="monkey-windows-64.exe"
 TRACEROUTE_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/traceroute64"
 TRACEROUTE_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/traceroute32"
 SAMBACRY_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/sc_monkey_runner64.so"
```

Comment:
We should do the same here with url's as in windows, but since these bins don't change the URL can stay the same I suppose.

Reply:
As mentioned in that thread, far harder to do this in Linux in a compatible manner.